envoy-control-tests/src/main/resources/lua_spec/ingress_rbac_logging_spec.lua

```diff
@@ -428,15 +504,27 @@ describe("envoy_on_response:", function()
 
         it("with globally allowed client", function ()
             -- given
-            metadata['envoy.filters.http.rbac']['shadow_engine_result'] = 'allowed'
+            metadata['envoy.filters.http.rbac']['shadow_engine_result'] = 'denied'
```

Comment:
Why did you change engine result? We should log requests for allowed client (allowed,denied).

Reply:
> Why did you change engine result?

No, this is an error in the unit test. The result is denied on master. See this comment: https://github.com/allegro/envoy-control/pull/243#discussion_r563514380